### ISSUE TITLE
out of date

### ISSUE THREAD

**Issue by MatthewSchimmel** [Muha0644/hoi4-dlc-mod#1]
This mod is not up to date with the latest focus trees

**Comment by Muha0644** [comment]
Yes, I am aware it's not up to date. That's why I said it only works with the old version.
I literally just took the vanilla files and replaced all dlc checks to always be true.

And it still doesn't work for some things, so there is no point in updating it.

 Now if I were to figure out a way to patch hoi4.exe to make SLCs work...
